source/include/ceph_troubleshooting.rst

```diff
@@ -1,3 +1,66 @@
+Investigating a Failed Ceph Drive
+---------------------------------
+
+After deployment, when a drive fails it may cause OSD crashes in Ceph.
+If Ceph detects crashed OSDs, it will go into `HEALTH_WARN` state.
+Ceph can report details about failed OSDs by running:
+
+.. code-block:: console
+
+   ceph# ceph health detail
+
+A failed OSD will also be reported as down by running:
+
+.. code-block:: console
+
+   ceph# ceph osd tree
+
+Note the ID of the failed OSD.
+
+The failed hardware device is logged by the Linux kernel:
+
+.. code-block:: console
+
+   storage-0# dmesg -T
+
+Cross-reference the hardware device and OSD ID to ensure they match.
+(Using `pvs` and `lvs` may help make this connection).
+
+Removing a Failed Ceph Drive
```

Comment:
That guide is for ceph-ansible only, right?

Reply:
It does seem so; therefore I'm not really in position to review it.

Reply:
Yes - I tried to split according to general operations and content specific to Ceph-Ansible.
I'm already looking forward to the CephAdm content!